docs/README.md

```diff
@@ -81,6 +81,10 @@ To access and use the review moderation admin interface, follow the instructions
 >
 > You can export reviews to XLS files from the **Download** tab, and it is limited to exporting 800 reviews at a time. Please use the date pickers to select the time range of reviews you want to export.
 
+> ⚠️ Warning
+>
+> If you don't see any review in this section and you see the **Migrate Data** button, you may have been using version 2.x on this account, you only need to click on this button to migrate all reviews saved during the v2, and then the page will be refreshed automatically, and the button will not appear any more.
```

Comment:
```suggestion
> If you see the **Migrate Data** button instead of a list of reviews in the **Catalog > Reviews** page, that means you are using an outdated version of the **Reviews and Ratings** app. To solve this issue and upgrade to the newer version of the app, click on **Migrate Data**. This will save all previous reviews in the newer version of the **Reviews and Ratings** app. Once the migration is finished, the page will automatically refresh, and the list of reviews will become available for you.
```